travis/install.sh

```diff
@@ -16,113 +16,158 @@ fi
 echo "NPROC=${NPROC}"
 export MAKEFLAGS="-j ${NPROC}"
 
-# prepare build files
-./autogen.sh
-
-# autoconf build
-mkdir build_autoconf
-cd build_autoconf
-../configure
-make dist-all >/dev/null
-# Check consistency of generated tarball
-TAR_FILENAME=`ls *.tar.gz`
-TAR_DIRECTORY=`basename $TAR_FILENAME .tar.gz`
-tar xvzf $TAR_FILENAME
-cd ..
+# Use ccache if it's available
+if command -v ccache &> /dev/null
+then
+    USE_CCACHE=ON
+    ccache -s
+else
+    USE_CCACHE=OFF
+fi
 
-# compare with CMake's dist
-mkdir build_cmake
-cd build_cmake
-cmake -DBUILD_TESTING=OFF ..
+if test "x${CMAKE_BUILD_TYPE}" = "x"; then
+    CMAKE_BUILD_TYPE=Release
+fi
+
+echo "Make dist tarball, and check consistency"
+mkdir build_dist
+cd build_dist
+cmake -D BUILD_TESTING=OFF ..
 make dist
-tar xzf $TAR_FILENAME
-cd ..
-diff -qr build_autoconf/$TAR_DIRECTORY build_cmake/$TAR_DIRECTORY || true
 
-# continue build from autoconf
-cd build_autoconf/$TAR_DIRECTORY
+TAR_FILENAME=$(ls *.tar.gz)
+TAR_DIRECTORY=$(basename $TAR_FILENAME .tar.gz)
+mkdir ../build_from_dist
+cd ../build_from_dist
+tar xvzf ../build_dist/$TAR_FILENAME
```

Comment:
you seem to hit on Travis-CI the same weirdness I got  with this tar command having apparently no effect... As I couldn't figure out what was going on I just disable building from the tar on Travis-CI : https://github.com/OSGeo/PROJ/pull/3020/files#diff-c47231c4caf6347330276d1d8abd9f76d5299bd4767e395dabb4f95755b0ce8fR19 and https://github.com/OSGeo/PROJ/pull/3020/files#diff-c47231c4caf6347330276d1d8abd9f76d5299bd4767e395dabb4f95755b0ce8fR177

Reply:
strange! I'd need to re-arrange the source dir for the other tests.